src/plugins/controls/public/controls/timeslider_control/components/time_slider_popover_content.tsx

```diff
@@ -27,29 +29,62 @@ interface Props {
   compressed: boolean;
 }
 
-export function TimeSliderPopoverContent(props: Props) {
-  const rangeInput = props.isAnchored ? (
+export function TimeSliderPopoverContent({
+  isAnchored,
+  setIsAnchored,
+  value,
+  onChange,
+  stepSize,
+  ticks,
+  timeRangeMin,
+  timeRangeMax,
+  compressed,
+}: Props) {
+  const [displayedValue, setDisplayedValue] = useState<Timeslice>(value ?? [0, 0]);
+
+  const debouncedOnChange = useMemo(
+    () =>
+      debounce((updateTimeslice: Timeslice | undefined) => {
+        onChange(updateTimeslice);
+      }, 350),
```

Comment:
I think we could make the debounce even higher now that we have the `onMouseUp` cancelling effect

```suggestion
      }, 750),
```